lints.toml

```diff
@@ -25,11 +25,14 @@ deny = [
 ]
 allow = [
     'clippy::as_conversions',
+    'clippy::cognitive_complexity',
```

Comment:
Btw added this lint to allowed also. Don't know how it is working though.

Reply:
It actually complained about the iterator over connection.

Reply:
Seems like its some formula of function complexity based on number of variables, conditions, etc.

Reply:
But yes, its bad that they do not specify the actual algorithm - it is probably due to the fact that it is quite controversial

Reply:
Yeah, seems like so:
https://github.com/rust-lang/rust-clippy/blob/6ae0835df01f51c75b516b02ab2983576cf6b0b3/clippy_lints/src/cognitive_complexity.rs